### PR TITLE
added 2 getting started articles

### PR DESCRIPTION
**Pull request by kouks**

### CODE REVIEW

File: articles/directory-structure.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Each of these directories includes 2 files and a folder by default.
+
+- **`main.ts`** is the entry point of the respective script. This file is
+provided by Exteranto and should bot be changed unless strictly necessary. It
+boots the framework, registers listeners and loads application config.
+
+- **`events.ts`** is the application "router" and event listener registrar in
+one. The file is used to register listeners to various events&mdash;both
+cross-script and withtin a script (bakground/content). Read more about how
```
Comment (haydnba) [inline articles/directory-structure.md]: within

**Comment by porkbrain** [comment]
Going to review today when I get home.

File: articles/directory-structure.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+events and messages work in the [Typed Message Passing][typed-message-passing]
+article.
+
+- **`/listeners`** is a default folder to store script-specific listeners. A
+simple listener is created and registered by Exteranto for demonstration. More
+about listeners in the previously mentioned article.
```
Comment (haydnba) [inline articles/directory-structure.md]: think it might be good to have the link anyway here?

**Review by haydnba — APPROVED** [review]

File: articles/directory-structure.md (hunk 2 above)
Comment (kouks) [reply articles/directory-structure.md]: yes

File: articles/directory-structure.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+#### `/static`
+
+A space dedicated to store your static assets, like images or documents. These
+assets can then be accessed from the codebase via the [`Extension`][Extension]
+APIs.
+
+#### `/test`
+
+The `test` directory contains a unit test scaffold based on [`mocha`][mocha].
```
Comment (porkbrain) [inline articles/directory-structure.md]: Probably not restricted to unit tests only, I'd just say "a test scaffold".

File: articles/directory-structure.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+---
+
+### The `/src` directory
+
+#### `/app`
+
+The `app` folder contains all main application logic that **needs to be split**
```
Comment (porkbrain) [inline articles/directory-structure.md:49]: What about "needs to be separated"?

File: articles/directory-structure.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+between the _background_ script and the _content_ script&mdash;hence the only
+two subfolders are `app/background` and `app/content`.
+
+Each of these directories includes 2 files and a folder by default.
+
+- **`main.ts`** is the entry point of the respective script. This file is
+provided by Exteranto and should bot be changed unless strictly necessary. It
```
Comment (porkbrain) [inline articles/directory-structure.md]: _typo_ "not be changed"

Comment (porkbrain) [reply articles/directory-structure.md]: "unless you know what you're doing" xd

File: articles/directory-structure.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+boots the framework, registers listeners and loads application config.
+
+- **`events.ts`** is the application "router" and event listener registrar in
```
Comment (porkbrain) [inline articles/directory-structure.md]: what about "is both the ..."

File: articles/directory-structure.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+[Typed Message Passing][typed-message-passing] article.
+
+#### `/assets`
+
+This folder contains assets that need to be compiled with the source code.
+Exteranto supports `.sass` styling by default.
+
+#### `/exceptions`
+
+The `exceptions` directory contains global definitions for exteption classes.
```
Comment (porkbrain) [inline articles/directory-structure.md]: "exteption" possibly typo

File: articles/directory-structure.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+more about how messages work in the
+[Typed Message Passing][typed-message-passing] article.
+
+#### `/providers`
+
+This folder contains service provider classes for registering custom services or
+events. Read more about these in the [Service Providers][service-providers]
+article.
+
+[&laquo; Previous (Installation)][installation]
```
Comment (porkbrain) [inline articles/directory-structure.md:91]: Nice :3

Comment (kouks) [reply articles/directory-structure.md:91]: ikr

File: articles/directory-structure.md (hunk 7 above)
Comment (kouks) [reply articles/directory-structure.md]: 50% typo

File: articles/directory-structure.md (hunk 4 above)
Comment (kouks) [reply articles/directory-structure.md:49]: eh doesn't really go well with the end of the sentence

**Pull request merged**